include/pka_config.h

```diff
@@ -75,11 +72,14 @@
 #define PKA_WINDOW_RAM_RING_MEM_SIZE         0x0800 //  2KB
 #define PKA_WINDOW_RAM_DATA_MEM_SIZE         0x3800 // 14KB
 
+#define PKA_WINDOW_RAM_OFFSET_MASK_BF1       0x730000
+#define PKA_WINDOW_RAM_OFFSET_MASK_BF2       0x730000
+
 // Macro for mapping PKA Ring address into Window RAM address. It converts the
 // ring address, either physical address or virtual address, to valid address
 // into the Window RAM. This is done assuming the Window RAM base and size.
-#define PKA_RING_MEM_ADDR(addr, size) \
-    (PKA_WINDOW_RAM_BASE | (((addr) & 0xffff) | \
+#define PKA_RING_MEM_ADDR(base, wndw_ram_off_mask, addr, size) \

```

Comment:
wndw_ram_off_mask is a long name for a macro, please use a short name as "mask" and add a comment to describe the arguments if needed.

Reply:
Will do, thanks.